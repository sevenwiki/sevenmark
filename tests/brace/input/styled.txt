{{{ #style="color:red" #size="16px" Styled text content }}}