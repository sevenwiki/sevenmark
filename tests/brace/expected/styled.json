[
  {
    "StyledElement": {
      "location": {
        "start": 0,
        "end": 59
      },
      "parameters": {
        "size": {
          "location": {
            "start": 23,
            "end": 36
          },
          "key": "size",
          "value": [
            {
              "Text": {
                "location": {
                  "start": 30,
                  "end": 34
                },
                "content": "16px"
              }
            }
          ]
        },
        "style": {
          "location": {
            "start": 3,
            "end": 23
          },
          "key": "style",
          "value": [
            {
              "Text": {
                "location": {
                  "start": 12,
                  "end": 21
                },
                "content": "color:red"
              }
            }
          ]
        }
      },
      "content": [
        {
          "Text": {
            "location": {
              "start": 36,
              "end": 56
            },
            "content": "Styled text content "
          }
        }
      ]
    }
  }
]